h2/src/tools/org/h2/build/Build.java

```diff
@@ -358,9 +366,21 @@ private void downloadOrVerify(boolean offline) {
         downloadOrVerify("ext/servlet-api-3.1.0.jar",
                 "javax/servlet", "javax.servlet-api", "3.1.0",
                 "3cd63d075497751784b2fa84be59432f4905bf7c", offline);
-        downloadOrVerify("ext/lucene-core-3.6.2.jar",
-                "org/apache/lucene", "lucene-core", "3.6.2",
-                "9ec77e2507f9cc01756964c71d91efd8154a8c47", offline);
+        downloadOrVerify("ext/lucene-core-5.5.5.jar",
+                "org/apache/lucene", "lucene-core", "5.5.5",
+                "c34bcd9274859dc07cfed2a935aaca90c4f4b861", offline);
+        downloadOrVerify("ext/lucene-analyzers-common-5.5.5.jar",
+                "org/apache/lucene", "lucene-analyzers-common", "5.5.5",
+                "e6b3f5d1b33ed24da7eef0a72f8062bd4652700c", offline);
+        downloadOrVerify("ext/lucene-queryparser-5.5.5.jar",
+                "org/apache/lucene", "lucene-queryparser", "5.5.5",
+                "6c965eb5838a2ba58b0de0fd860a420dcda11937", offline);
+        downloadOrVerify("ext/lucene-queries-5.5.5.jar",
+                "org/apache/lucene", "lucene-queries", "5.5.5",
+                "d99719e7c58c149113f897bca301f1d68cbf3241", offline);
+        downloadOrVerify("ext/lucene-sandbox-5.5.5.jar",
```

Comment:
Do we really need `lucene-queries` and `lucene-sandbox` here? I didn't found where they are used.

Reply:
I have no idea, probably not, and it is a pain in the neck to maintain those, but that's what is listed in pom.xml for lucene-queryparser as dependencies. :disappointed: 
Do we really need a QueryParser? For pure text search without wildcards, regular expressions etc. - we do not, and then just lucene-core and lucene-analyzers-common is enough.

Reply:
There are some differences between versions of Lucene. lucene-queries is needed with recent versions of Lucene (probably since 6.5.0 or 6.5.1), but not with 5.5.5. Because our test suit uses 5.5.5 I think that there is no reason to download `lucene-queries` in it.

lucene-sandbox most likely is not needed at all.

H2 uses lucene-queryparser. Without `StandardQueryParser` queries with `AND` and similar constructions will not work any more.